File: AtCoder/abc200/C.swift
var s = readLine() ?? ""
let n = Int64(s) ?? Int64(0)
let arr = readLine()?.split(separator: " ").map({ Int64($0) ?? Int64(0) }) ?? []

var mp = [Int64:Int64]()

arr.forEach { ele in
	let key = (ele + 200) % 200
	let value = mp[key] ?? Int64(0)
	mp[key] = value + 1
}

var ans = Int64(0)

mp.forEach { (key, value) in
	ans += (value * (value - 1)) / 2
}

print(ans)

File: AtCoder/abc200/C.kt
fun main() {
    val n = readLine()?.toLong() ?: 0L
    val mp = hashMapOf<Long, Long>()
    val arr = readLine()?.split(" ")?.map { it.toLong() } ?: emptyList()
    arr.forEach {
        val key = (it + 200) % 200
        mp[key] = mp.getOrDefault(key, 0L) + 1
    }
    var ans = 0L
    mp.forEach { (_, value) ->
        ans += (value * (value - 1)) / 2
    }
    print(ans)
}
